iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/repository/HouseTableRepositoryImpl.java

```diff
@@ -39,12 +39,15 @@
 public class HouseTableRepositoryImpl implements HouseTableRepository {
 
   /**
-   * The request timeout is decided based on retry template logic and server side gateway timeout of
-   * 60 sec. The retry template has retry max attempt of 3 with 2 secs delay (with delay multiplier
-   * as attempt increases) between each retry. So the overall retry process should complete within
-   * 60 sec.
+   * The read request timeout is decided based on retry template logic and server side gateway
```

Comment:
can we go ahead and add jitter to this request to prevent thread synchronization? 

due to recent mysql outage.

Reply:
lets not block this PR. This is P0 to avoid corruption issues. Adding jitter can be done in a separate one.

Reply:
ok.

for future reference:

```
public static final BackOffPolicy DEFAULT_HTS_BACKOFF_POLICY =
      RetryTemplate.builder()
            .maxAttempts(MAX_RETRY_ATTEMPT)
            .exponentialBackoff(2000L, 2.0, 30000, true)
            .retryOn(throwables)
            .build();
}
```